diesel/src/expression/not.rs

```diff
@@ -0,0 +1,72 @@
+use expression::*;
+use query_builder::*;
+use result::QueryResult;
+use types::Bool;
+
+/// Creates a SQL `NOT` expression
+///
+/// # Example
+///
+/// ```rust
+/// # #[macro_use] extern crate diesel;
+/// # include!("src/doctest_setup.rs");
+/// #
+/// # table! {
+/// #     users {
+/// #         id -> Integer,
+/// #         name -> VarChar,
+/// #     }
+/// # }
+/// #
+/// # fn main() {
+/// #     use self::users::dsl::*;
+/// #     let connection = establish_connection();
+/// use diesel::expression::dsl::not;
```

Comment:
Why not `use diesel::expression::not` ? That's what rust told me to use when it was missing